MaxText/accelerator_to_spec_map.py

```diff
@@ -150,6 +151,8 @@ class SystemCharacteristics:
     "v5p-12288": SystemCharacteristics("tpu", "v5:16x16x24", "megacore", (2, 2, 1), 6144, (True, True, True)),
     "v5p-13824": SystemCharacteristics("tpu", "v5:12x24x24", "megacore", (2, 2, 1), 6912, (True, True, True)),
     "v5p-17920": SystemCharacteristics("tpu", "v5:16x20x28", "megacore", (2, 2, 1), 8960, (True, True, True)),
+    # A3 - H100
+    "a3": SystemCharacteristics("gpu", "a3:8", None, None, 8, None)
```

Comment:
Is `topology_name` field needed for A3? I didn't see it being used in the implementation. How about marking it as Optional and setting it as None by default? I'm just afraid that setting it as "a3:8" by default will provide an impression that only 8 A3 GPUs are being supported. 

Besides, it'd be great if you could add a comment about setting `compile_topology_num_slices` if users want to run for multi hosts (I see you added it in the doc below, but it doesn't hurt to also add in the code:)).

Reply:
Sure, great callouts! I'll add a comment to the code as well.

Since A3's topology only supports a single configuration, this field also isn't necessary. I'll make the field optional.